### DISCUSSION TITLE
In 01_workflow where does the code save the model?

### DISCUSSION THREAD

**Discussion by rwkopcke** [mrdbourke/pytorch-deep-learning#1105] [Q&A] [answered]
`MODEL_PATH = Path("models")
MODEL_PATH.mkdir(parents=True, exist_ok=True)`
With this code in a colab cell, where do I find the directory *models*?

**Comment by AI-Titans** [comment, marked as answer]
You can find the models directory in the File Explorer panel on the left side of the Google Colab interface. Alternatively, you can run `!ls` in a code cell to list the contents of the current directory and see the models folder there
